pm/sendermonitor.go

```diff
@@ -180,6 +184,19 @@ func (sm *senderMonitor) QueueTicket(addr ethcommon.Address, ticket *SignedTicke
 	sm.senders[addr].queue.Add(ticket)
 }
 
+// IsLocked checks whether a sender's unlock period ends the round after the next round
+func (sm *senderMonitor) IsLocked(addr ethcommon.Address) error {
+	info, err := sm.smgr.GetSenderInfo(addr)
+	if err != nil {
+		return fmt.Errorf("could not get sender info for %v: %v", addr.Hex(), err)
+	}
+	maxWithdrawRound := new(big.Int).Add(sm.rm.LastInitializedRound(), big.NewInt(1))
```

Comment:
Potentially, I see `maxWithdrawRound` as more general in case we deem the current delta to not be sufficient but in this particular case `nextRound` is more descriptive.